File: SustitucionMOA/src/app/common/models/log-table.ts
export interface LogTableDto {
    Id: number;
    Date: string;
    Level: string;
    Logger: string;
    Message: string;
    Exception: string;
}

export interface LogTableCountErrors {
    Logger: string;
    Level: string;
    LoggerLevel: string;
    Count: number;
    Errors: LogTableDto[];
}

File: SustitucionMOAModel/Entities/LogTable.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace SustitucionMOAModel.Entities
{
    [Table("LogTable", Schema = "logs")]
    public class LogTable
    {
        [Key]
        public long Id { get; set; }
        public DateTime Date { get; set; }
        public string Level { get; set; }
        public string Logger { get; set; }
        public string Message { get; set; }
        public string Exception { get; set; }
    }
}
